tests/cli_commands/test_init.py

```diff
@@ -29,15 +29,16 @@ class InitCommandTestCase(unittest.TestCase):
             ),
             "local-template": TemplateConfig(
                 template_name="local-template",
-                template_path="/var/tmp/Documents/project-template",
+                template_path="/Users/test_user/Documents/project-template",
```

Comment:
does that mean every developer will have "test_user" dir under Users? Why do you need this change?

Reply:
`example_config_dict` contains data generated by user from [template](https://github.com/getindata/data-pipelines-cli/blob/bump-copier/tests/goldens/config_template/dp.yml.jinja), and in this test the answer for question `username` will be `test_user`. I kinda trimmed this to have only one question but the outcome is the same